Plugins/AssetUsageDetector/Editor/ObjectToSearch.cs

```diff
@@ -95,6 +95,21 @@ private void AddSubAssets( Object target, bool includeTarget, bool? shouldSearch
 					}
 				}
 
+				// If is an Atlas we get sprites directly as GetSubAssets return the final texture
+				if ( target is UnityEngine.U2D.SpriteAtlas spriteAtlas )
+				{
+					Sprite[] packedSprites = AssetUsageDetector.spriteAtlasPackedSpritesGetter( spriteAtlas );
+					if( packedSprites != null )
+					{
+						for( int i = 0; i < packedSprites.Length; i++ )
+						{
+							if ( currentSubAssets.Add( packedSprites[i] ) )
```

Comment:
I've found out that the returned Sprite can be null if it's deleted while the SpriteAtlas hasn't been repacked/rebuilt yet. So a null check is necessary here.